apis/v1alpha1/shared_types.go

```diff
@@ -181,24 +181,27 @@ type RouteGatewayStatus struct {
 	//
```

Comment:
> For example, the "Admitted" condition indicates whether the route has been admitted or rejected by the Gateway, and why.

Instead of the comment above, maybe we should say something like this::

> Controllers must include an "Admitted" condition to indicate whether the route has been admitted or rejected by the Gateway.

Reply:
> Controllers must include an "Admitted" condition to indicate whether the route has been admitted or rejected by the Gateway.

@robscott if we default the Admitted condition as described in https://github.com/kubernetes-sigs/service-apis/pull/528#discussion_r571154934, then I would expect the documentation to state:

> Controllers must update the "Admitted" condition to indicate whether the route has been admitted or rejected by the Gateway.